SOURCE/set_TopoBenchCase1.c

```diff
@@ -0,0 +1,353 @@
+#include "stdio.h"
+#include "stdlib.h"
+#include "string.h"
+#include "math.h"
+#include "header_MDOODZ.h"
+
+/*--------------------------------------------------------------------------------------------------------------------*/
+/*------------------------------------------------------ M-Doodz -----------------------------------------------------*/
+/*--------------------------------------------------------------------------------------------------------------------*/
+
+// SIMPLE
+void BuildInitialTopography( surface *topo, markers *topo_chain, params model, grid mesh, scale scaling ) {
```

Comment:
I don't see examples of set files where we use topo, model and mesh in this method.
Doesn't have to be fixed in this pull request, but we might add TODO here